jb/fftw/ut_time_delay_estimator_many.cpp

```diff
@@ -1178,3 +1180,35 @@ BOOST_AUTO_TEST_CASE(fftw_time_delay_estimator_many_vector_tde_complex_double) {
           confidence, expected_confidence, confidence_tol),
       "confidence is not within tolerance(" << confidence_tol << ")");
 }
+
+/**
+ * @test Verify thattime delay estimator handles errors
+ */
+BOOST_AUTO_TEST_CASE(fftw_time_delay_estimator_many_errors) {
```

Comment:
ACK. What do you want me to do now that you fixed the testing part. Should I continue pushing the existing one?